bin/docker-images-has-changed.py

```diff
@@ -38,15 +38,25 @@ def find_image_deps(image, cache):
 imgs = {}
 image_deps = {}
 for reponame in repos:
-  for img in get_docker_images(reponame):
+  images_list = get_docker_images(reponame)
+
+  # Generate a dictionary of base image checksum
+  base_dict = {}
+  for element in images_list:
+    name = element["BASE_DOCKER_REPOSITORY"]
+    arch = element["ARCHITECTURE"]
+    if name != "cmssw":
+        base_dict[arch] = element["BASE_UPSTREAM_CHECKSUM"]
```

Comment:
I think once we have found a non-cmssw container then we can break the loop ... right?

Reply:
Mmm... but we need to populate `base_dict` for all archs

Reply:
ah ok, right